src/validator.js

```diff
@@ -0,0 +1,64 @@
+import * as fs from 'fs';
+import Xpi from  'xpi';
+
+import promisify from 'es6-promisify';
+import chalk from 'chalk';
+
+export var lstat = promisify(fs.lstat);
+
+
+export default class Validator {
+
+  constructor(config) {
+    this.config = config;
+    this.packagePath = config._[0];
+    this.xpi;
+    this.chalk = new chalk.constructor(
+      {enabled: !this.config.boring});
+  }
+
+  handleError(err) {
+    if (this.config.stack === true) {
```

Comment:
This is a nitpick, but testing for truthiness here would seem to suffice, right?

Would you prefer always using the explicit `=== true` going forward? If that's how you did things on payments/you prefer it that's cool, just making notes for #21.